src/Component.js

```diff
@@ -41,11 +39,11 @@ export class CopyToClipboard extends React.PureComponent {
       text: _text,
       onCopy: _onCopy,
       options: _options,
+      onClickCapture,  // eslint-disable-line
       children,
-      ...props
+      ...restProps
     } = this.props;
-    const elem = React.Children.only(children);
 
-    return React.cloneElement(elem, {...props, onClick: this.onClick});
+    return React.cloneElement(children, {...restProps, onClick: this.onClick});
```

Comment:
Using just one child with regular react still works this way.

Many libraries out there don't work with array of children anyway. I understand your concern, I just think it's not going to cause much trouble.

However, for those using `preact-compat`, `React.Children` still works, so this change is optional. If the other part of this PR is fine, I can revert this to use `React.children.only`